pkg/operator/bootstrap.go

```diff
@@ -88,15 +88,15 @@ func RenderBootstrap(
 		return err
 	}
 	if additionalTrustBundleData != nil {
-		obji, err = runtime.Decode(configscheme.Codecs.UniversalDecoder(configv1.SchemeGroupVersion), additionalTrustBundleData)
+		obji, err := runtime.Decode(scheme.Codecs.UniversalDecoder(corev1.SchemeGroupVersion), additionalTrustBundleData)
```

Comment:
Curious, why is this file a configmap rather than just containing the ca-bundle data directly?

Reply:
The installer writes that CM to be installed and managed by some other operator (don't know the full rationale behind that but it's because it must be easier for the installer to just create the cm and reuse it here for the MCO and elsewhere)